app/styles/modules/_marketing-ios.scss

```diff
@@ -0,0 +1,45 @@
+.marketing-ios {
+  background-color: $html-background-color;
+  border-radius: 0 0 $big-border-radius $big-border-radius;
+  border-top: 1px solid $marketing-border-color;
+  box-shadow: 0 1px 1px $content-background-color inset;
+  font-size: $medium-font;
+  position: relative;
+
+  .os-general {
+    ul {
+      padding: 0;
+    }
+
+    li {
+      list-style-type: none;
+      display: block;
+      margin-bottom: 10px;
+    }
+  }
+
+  @include respond-to('small') {
+    margin: 20px -20px -20px;
+    font-size: $base-font;
```

Comment:
You might laugh, we've made it a rule to list properties in alphabetical order. Can you update this?